Makefile

```diff
@@ -180,10 +180,9 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 .PHONY: artifacts
 artifacts: kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMAGE_TAG}
+	if [ -d artifacts ]; then rm -rf artifacts ; fi
 	@mkdir -p artifacts
-	@rm -r artifacts/*
 	$(KUSTOMIZE) build config/default -o artifacts/manifests.yaml
-	@$(call clean-manifests)
```

Comment:
why remove this line?

Reply:
When cutting a release, we want to `kustomize set image` to stay, not to rewrite it back to `gcr.io` "staging" 
See release-0.1 the config/manager/kustomize goes to staging it should go to k8s.io or after this PR to registry.k8s

Reply:
What I did last time is actually update the value in the definition of `clean-manifests`.

But maybe what you propose is good.